app/frontend/app/templates/group/manage.hbs

```diff
@@ -0,0 +1,20 @@
+<div class="container">
+  <div class="col-sm-12">
+    {{#if currentMember.isAdmin}}
+        <div class="panel panel-default">
+          <div class="panel-heading">
+            Group Settings
+          </div>
+          <div class="panel-body">
+            {{#if model.closed}}
+              Your group is closed! You'll need to contact a staff member to re-open it.
+            {{else}}
+              <button class="btn btn-danger" {{action "closeGroup"}}>
+                Close Group
+              </button>
```

Comment:
If we're gonna maintain this page for site administrators then we should rephrase it to "Delete Group and all posts seriously don't fucking do this unless the group is evil"